src/main/java/report_screens/ReportScreen.java

```diff
@@ -58,12 +61,18 @@ public ReportScreen(ReportController controller, Review current_review, User cur
         report.addActionListener(this);
         cancel.addActionListener(this);
 
-        //JPanel settings
-        this.setLayout(new BoxLayout(this, BoxLayout.PAGE_AXIS));
-        this.setSize(200, 300);
-        this.add(title);
-        this.add(reasonInfo);
-        this.add(buttons);
+        JPanel main = new JPanel();
+        main.add(buttons);
+        main.add(reasonInfo);
+
```

Comment:
You should set main's layout to something like a box layout using main.setLayout()

Reply:
oh right, missed that part, thx